tests/TestCase/View/StringTemplateTest.php

```diff
@@ -440,4 +443,98 @@ public function testAddClassMethodUseIndex()
             'non-existent' => ['new_class']
         ]);
     }
+
+    /**
+     * Test removeClass method input parameter
+     *
+     * Tests null, string, array, false and object
+     *
+     * @return void
+     */
+    public function testRemoveClassMethodCurrentClass()
+    {
+        $result = $this->template->removeClass(['class' => ['keep_class', 'remove_class']], 'remove_class');
+        $this->assertEquals($result, ['class' => ['keep_class']]);
+
+        $result = $this->template->removeClass('keep_class remove_class', 'remove_class');
+        $this->assertEquals($result, ['class' => ['keep_class']]);
+
+        $result = $this->template->removeClass(null, 'remove_class');
+        $this->assertEquals($result, ['class' => []]);
+
+        $result = $this->template->removeClass(false, 'remove_class');
+        $this->assertEquals($result, ['class' => []]);
+
+        $result = $this->template->removeClass(new \StdClass(), 'remove_class');
```

Comment:
stdClass()

Reply:
@dereuromark, I updated this for both `addClass()` and `removeClass()`